src/main/java/io/rpg/Main.java

```diff
@@ -0,0 +1,41 @@
+package io.rpg;
+
+import io.rpg.util.Result;
+import javafx.application.Application;
+import javafx.stage.Stage;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.config.Configurator;
+
+import java.io.IOException;
+
+public class Main extends Application {
+  @Override
+  public void start(Stage stage) throws IOException {
+    Configurator.setRootLevel(Level.DEBUG);
+    Logger logger = LogManager.getLogger(Main.class);
+
+    Initializer worldInitializer = new Initializer("configurations/config-1", stage);
+    Result<Game, Exception>  initializationResult = worldInitializer.initialize();
+
+    if (initializationResult.isError()) {
+      logger.error("Initialization error");
+      // TODO: HANLDE BAD INIT
+    } else if (initializationResult.isOkValueNull()) {
+      // TODO: HANDLE
+    }
+
+    Game game = initializationResult.getOkValue();
+
+    stage.setScene(game.getWorldView());
+
+    stage.show();
+    System.out.println("SCENE");
+    System.out.println(stage.getScene());
+  }
+
+  public static void main(String[] args) {
```

Comment:
Parametry są przekazywane nawet bez tej funkcji

Reply:
Nie do końca rozumiem. Wydaje mi się, że żeeby dostać się do argumentów wywołania musimy zrobić wrapper na tą klasę (inny entry point do aplikacji). Tylko coś mi się kojarzy że JavaFX wymaga, żeby entry point aplikacji dziedziczył z `Application`... Ale to jest jeszcze do sprawdzenia.

Reply:
Już rozumiem, znalazłem -- JavaFX opakowuje parametry wywołania i są później dostępne przez `getParameters()`. To tylko lepiej dla nas.